Modules/_collectionsmodule.c

```diff
@@ -65,173 +70,231 @@ class dequeobject_converter(self_converter):
 /*[python end generated code: output=da39a3ee5e6b4b0d input=b6ae4a3ff852be2f]*/
 
 /* collections module implementation of a deque() datatype
-   Written and maintained by Raymond D. Hettinger <python@rcn.com>
+   Originally written by Raymond D. Hettinger <python@rcn.com>
```

Comment:
@rhettinger I'm happy to handle this however you feel like, including leaving the whole comment as is.